src/FluentMigrator.DotNet.Cli/Commands/ConnectionCommand.cs

```diff
@@ -25,17 +25,20 @@ namespace FluentMigrator.DotNet.Cli.Commands
     public class ConnectionCommand : MigrationCommand
     {
         [Option("-c|--connection <CONNECTION_STRING_OR_NAME>", Description = "The name of the connection string (falls back to machine name) or the connection string itself to the server and database you want to execute your migrations against.")]
-        public string ConnectionString { get; }
+        public string ConnectionString { get; set; }
+
+        [Option("-m|--transaction-mode <MODE>", Description = "Overrides the transaction behavior of migrations, so that all migrations to be executed will run in one transaction.")]
+        public TransactionMode TransactionMode { get; set; }
```

Comment:
This is a good change and I'd like to keep this in the PR.

Reply:
@fubar-coder My only thought is that the "mode" is a change from the switches provided by the Console Runner which uses `--transaction-per-session`.  https://fluentmigrator.github.io/articles/runners/runner-console.html#--transaction-per-session---tps-optional

Perhaps the short-hand should be `-tm`.

To be clear, I think specifying a transaction mode is way better than a switch, as its future proof.  As i've mentioned in the past, I want to add support for monadic transactions.